src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -497,7 +497,8 @@ const getServices = (state: RootState): Services | undefined => {
   if (
     services.enabled.length === 0 &&
     services.masked.length === 0 &&
-    services.disabled.length === 0
+    services.disabled.length === 0 &&
+    !selectFirstBootScript(state)
```

Comment:
nit: instead of making this condition more complex, we can change the order and use the `enabledSvcs` in the condition:

```suggestion
  const enabledSvcs = services.enabled || [];
  const includeFBSvc: boolean =
    !!selectFirstBootScript(state) &&
    !services.enabled?.includes(FIRST_BOOT_SERVICE);
  if (includeFBSvc) {
    enabledSvcs.push(FIRST_BOOT_SERVICE);
  }
  if (
    enabledSvcs.length === 0 &&
    services.masked.length === 0 &&
    services.disabled.length === 0
  ) {
    return undefined;
  }
```

Reply:
Thanks @amirfefer! I also though about this, but first wanted to check if I solved the problem in the first place!
I'll revise